abci/server/socket_server.go

```diff
@@ -38,25 +42,25 @@ func NewSocketServer(protoAddr string, app types.Application) service.Service {
 		app:      app,
 		conns:    make(map[int]net.Conn),
 	}
-	s.BaseService = *service.NewBaseService(nil, "ABCIServer", s)
+	// Use standard logger by default.
+	logger := log.NewStdLibLogger(os.Stdout, "abci_server", stdlog.LstdFlags)
```

Comment:
Let's not set a default logger, this breaks with conventions elsewhere in Tendermint. I think a simple `fmt.Printf()` call in the panic handler if the logger is `nil` should be sufficient, this is a last-ditch fallback more than anything.

Reply:
Yeah, I agree. I'll make changes